apps/zotonic_core/src/install/z_install.erl

```diff
@@ -389,6 +389,30 @@ model_pgsql() ->
     "CREATE UNIQUE INDEX IF NOT EXISTS identity_type_key_unique ON identity (type, key) WHERE (is_unique)",
     "CREATE INDEX IF NOT EXISTS identity_type_key_key ON identity using btree (type, key collate ucs_basic text_pattern_ops)",
 
+    % Table identity log
+    % Keeps track of all successful logons of a user (for some time)
+
+    "CREATE TABLE IF NOT EXISTS identity_log
+    (
+      id serial NOT NULL,
+      identity_id int NOT NULL,
+      rsc_id int NOT NULL,
+      created timestamp with time zone NOT NULL DEFAULT now(),
+      user_agent character varying (240),
+      ip_address character varying (40),
+
+      CONSTRAINT id_log_pkey PRIMARY KEY (id),
+      CONSTRAINT pk_id_log_rsc_id FOREIGN KEY (rsc_id)
```

Comment:
We usually name these foreign key constraints `fk_...` and not `pk_...` 

As `identity_id` has a foreign key constraint we also need an index on that columns to prevent table scans on deletion of an identity table row.

Reply:
Makes sense, fixed in 12c52c56b165968cdafcc47727eac3aa06924584